packages/model/src/reminder.ts

```diff
@@ -46,7 +46,7 @@ export interface ResearchOutputPublishedReminder
     researchOutputId: ResearchOutputDataObject['id'];
     documentType: ResearchOutputDataObject['documentType'];
     title: ResearchOutputDataObject['title'];
-    addedDate: ResearchOutputDataObject['addedDate'];
+    addedDate: string;
```

Comment:
?

Reply:
this is valid but better go with 
```suggestion
    addedDate: NonNullable<ResearchOutputDataObject['addedDate']>;
```

Reply:
or just use the PublishedResearchOutputDataObject